x-pack/plugins/index_lifecycle_management/__jest__/policy_table.test.tsx

```diff
@@ -132,10 +136,10 @@ describe('policy table', () => {
     const perPageButton = rendered.find('EuiTablePagination EuiPopover').find('button');
     perPageButton.simulate('click');
     rendered.update();
-    const fiftyButton = rendered.find('.euiContextMenuItem').at(1);
-    fiftyButton.simulate('click');
+    const numberOfRowsButton = rendered.find('.euiContextMenuItem').at(1);
```

Comment:
nit: I'm not able to add a comment on the correct line, but I think the test description could be improved. Took me a little bit to understand what this was testing - maybe something like:

```
should show more policies when the "Rows per page" value is increased
```